src/HtmlProcessor/CssToAttributeConverter.php

```diff
@@ -164,11 +165,12 @@ private function mapSimpleCssProperty(string $property, string $value, \DOMEleme
         }
 
         $mapping = $this->cssToHtmlMap[$property];
-        $nodesMatch = !isset($mapping['nodes']) || \in_array($node->nodeName, $mapping['nodes'], true);
-        $valuesMatch = !isset($mapping['values']) || \in_array($value, $mapping['values'], true);
+        $nodesMatch = !isset($mapping['nodes']) || \in_array($node->nodeName, (array)$mapping['nodes'], true);
+        $valuesMatch = !isset($mapping['values']) || \in_array($value, (array)$mapping['values'], true);
```

Comment:
If we can be more explicit in the type hint for `$cssToHtmlMap`, the cast wouldn't be needed - ?

Reply:
Done.